### PR TITLE
manage gradle ignoreFailures in GH Actions and TC build

### PR DESCRIPTION
**Pull request by conker84**
manage gradle ignoreFailures in GH Actions and TC build

### CODE REVIEW

**Review by fbiville — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -86,7 +86,10 @@ subprojects {
             setFailOnNoMatchingTests(false)
         }
 
-        ignoreFailures(true)
+        // in TC we don't have the CI env so we can manage it in this way
```
Comment (fbiville) [inline build.gradle:89]: that will also affect local builds, but CI on GH actions will at least catch it, so I guess it's not too bad

**Review by fbiville — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (fbiville) [reply build.gradle:89]: maybe there is a specific envvar for TC? That would be better.

Comment (conker84) [reply build.gradle:89]: Thank you so much! About the local build I was aware of it and I agree with, the only thing is that maybe we can try to leverage instead some TC parameter but I preferred to **play it safe**

Comment (fbiville) [reply build.gradle:89]: `TEAMCITY_VERSION` seems like a good bet: https://www.jetbrains.com/help/teamcity/predefined-build-parameters.html

Comment (conker84) [reply build.gradle:89]: oki

**Pull request merged**